docs/en/notes/faq.md

```diff
@@ -26,6 +26,11 @@ We list some common troubles faced by many users and their corresponding solutio
   1. Uninstall existing `mmcv-lite` in the environment using `pip uninstall mmcv-lite`.
   2. Install `mmcv` following the [installation instruction](https://mmcv.readthedocs.io/en/2.x/get_started/installation.html).
 
+- "Microsoft Visual C++ 14.0 or graeter is required" during installation on Windows.
```

Comment:
Will this happen in installing pycocotools?

Reply:
yes, `building 'pycocotools._mask' extension`.

Reply:
Then can we indicate this in the FAQ so that readers could better understand the cause?

Reply:
> Then can we indicate this in the FAQ so that readers could better understand the cause?

OK, I have added it.